### PR TITLE
Implemented topoSort and added skeleton for areaOfConvexPolygon.

### LINKED ISSUE #361: Fix topoSort method
There is currently a method called `topoSort` that lives in `/lib/topoSort.js`.
It is incomplete and needs to be fixed!

It [topologically sorts](https://en.wikipedia.org/wiki/Topological_sorting) a directed graph represented by its adjacency list.

### Acceptance Criteria:
1. Running `yarn test topoSort` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by Ampretuzo**
Closes #361 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: test/areaOfConvexPolygon.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      [1, 0],
+      [1, 1],
+      [0.5, 2],
+      [0, 1]
+    ])).toBeCloseTo(1.5000000, 4);
+  });
+
+  test("simple 6 sided figure: [[0, 0], [1, 0], [1, 1], [0.75, 2], [0.25, 2], [0, 1] ] -- area must be 1.75", () => {
+    expect(areaOfConvexPolygon([
+      [0, 0],
```
Comment (devanp92) [inline test/areaOfConvexPolygon.test.js]: I think this test case needs to updated with the correct array stated in the description.

**Comment by BlakeGuilloud** [comment]
Hey @Ampretuzo - if you could please resolve the current merge conflicts, we can get this in! Thanks

**Comment by Ampretuzo** [comment]
@BlakeGuilloud traverse method depends on added metadata to the graph argument, so probably no. I put it there for readability actually.

btw, how did it come my skeleton implementation was merged first? Not good, not good :)

**Comment by ktilcu** [comment]
@Ampretuzo thanks for the contribution. We have a lot of PRs so if a skeleton method is in a separate PR then we probably took it at face value and merged it as a help. I merged this one without a skeleton because of that. Happy Hacktobering.

**Pull request merged**